### PR TITLE
Add hunterling merc

### PR DESCRIPTION
**Pull request by itsjustbones**
Adds the hunterling merc, is under the sc2/modern tab of predator nest, based on requests, and possibly for use in devil's playground raceswap
I reduced some stats from the coop outbreak map version
stun time from 5 -> 3 seconds
Leap cooldown 4 -> 10 seconds
leap damage 100 -> 20 damage
I also doubled the leap speed, since it's very slow to give time for players to dodge in the mission.
I applied 3 zergling upgrades to them, shredding claws, adrenal overload, and metabolic boost.

### CODE REVIEW

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml [hunk 1]
```diff
@@ -35250,4 +35250,98 @@
...
+        <On Terms="Abil.AP_LeapAttack.SourceCastStart" Send="AnimClear Jump"/>
+        <On Terms="Abil.AP_LeapAttack.SourceCastStart" Send="AnimBracketStart LeapAttack Jump,Start Jump IGNORE"/>
+        <On Terms="Effect.AP_HunterlingLeapImpactDamageSearch.Start; At Caster" Send="AnimBracketStop LeapAttack"/>
+        <On Terms="Effect.AP_LeapHunterlingDamage.Start; At Caster" Send="AnimPlay PounceAttacking Spell"/>
+        <On Terms="UnitHeightUpdate.*.Land" Send="Create AP_Hunterling_Land"/>
+        <!--On Terms="Effect.LeapNoTargetDummy.Start; At Caster" Send="AnimPlay Land Jump,End,A"/>
+        <On Terms="Effect.LeapAttackDelayPersistent.Start; At Caster" Send="AnimPlay Falling Jump,End 0 0.000000 0.000000"/>
+        <On Terms="Effect.LeapAttackingPersistent.Start; At Caster" Send="AnimPlay PounceAttacking Spell PlayForever"/>
+        <On Terms="Behavior.LeapAttacking.Off" Send="AnimClear PounceAttacking"/-->
+        <Macros value="AntiGravDeathRemoveMacro"/>
```
Comment (Ziktofel) [inline Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml:35307]: Add unit response sounds

**Review by Ziktofel — CHANGES_REQUESTED** [review]
Missing attack hit sound

**Comment by Ziktofel** [comment]
Also update merc-related upgrades

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml (hunk 1 above)
Comment (itsjustbones) [reply Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml:35307]: sorry, I forgot to sync one of my commits to github, which fixed sounds and some other things

**Comment by itsjustbones** [comment]
PR was originally missing a commit since I didn't sync it to github by accident

**Comment by itsjustbones** [comment]
> Also update merc-related upgrades

This should be fixed

**Pull request merged**